docs/architecture/joins.txt

```diff
@@ -26,13 +26,12 @@ elements appear in which join.
 
 .. _joins_figure_1:
 
-.. figure:: joins.png
-   :scale: 70%
+.. figure:: joins.jpeg
    :align: center
 
-   Figure 1
+   Join Types
 
-   LEFT JOIN, RIGHT JOIN, INNER JOIN, OUTER JOIN, and CROSS JOIN of a set L and
+   From left to right, top to bottom: left join, right join, inner join, outer join, and CROSS JOIN of a set L and
```

Comment:
IMHO I would keep the join types with capital letters, as CROSS JOIN is

Reply:
hmm. in the documentation I am trying to keep a separation between `LEFT JOIN` as a specific piece of SQL syntax, an "left join" as a concept

does this make sense?

Reply:
I don't insist, it's just my preference, at least then make the cross join lowercase as well.

Reply:
This is not fixed (at least make CROSS JOIN lowercase)....

Reply:
@nslater CROSS JOIN is still uppercase...